segmentation/include/pcl/segmentation/conditional_euclidean_clustering.h

```diff
@@ -106,6 +106,24 @@ namespace pcl
       {
       }
 
+      /** \brief Provide a pointer to the search object.
+        * \param[in] tree a pointer to the spatial search object.
+        */
+      inline void 
+      setSearchMethod (const SearcherPtr &tree) 
+      { 
+        searcher_ = tree; 
+      }
+
+      /** \brief Get a pointer to the search method used. 
+       *  @todo fix this for a generic search tree
```

Comment:
Can you provide some more info on this comment?

Reply:
actually this was just copy/pasted from here:
https://github.com/PointCloudLibrary/pcl/blob/master/segmentation/include/pcl/segmentation/extract_clusters.h#L318

Reply:
Shall I just remove the `@todo` line? I don't see why the brief description would not be sufficient here.

Reply:
Yup let's go with that.